samples/multi-module/CMakeLists.txt

```diff
@@ -49,6 +49,12 @@ endif ()
 if (NOT DEFINED WAMR_BUILD_JIT)
   set(WAMR_BUILD_JIT 0)
 endif ()
+if (NOT DEFINED WAMR_BUILD_DUMP_CALL_STACK)
+  set(WAMR_BUILD_DUMP_CALL_STACK 1)
```

Comment:
Had better set it 0 and use `cmake -DWAMR_BUILD_DUMP_CALL_STACK=1` to enable it if needed?